<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Disk</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="version">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>113</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Label</string>
    </property>
   </widget>
   <widget class="QListWidget" name="labelField">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>111</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="nameField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="package_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>113</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>113</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Free</string>
    </property>
   </widget>
   <widget class="QListWidget" name="freeField">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>111</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>113</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QListWidget" name="sizeField">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>111</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
